ftwhttp/request.go

```diff
@@ -127,7 +134,7 @@ func BuildRequest(r *Request) ([]byte, error) {
 	}
 
 	// Multipart form data needs to end in \r\n, per RFC (and modsecurity make a scene if not)
-	if ct := r.headers.Value(ContentTypeHeader); strings.HasPrefix(ct, "multipart/form-data;") {
+	if r.headers.HasAnyValueContaining(header_names.ContentType, "multipart/form-data;") {
 		crlf := []byte("\r\n")
```

Comment:
We have a constant for headerSeparator, maybe we want to use it here? https://github.com/coreruleset/go-ftw/pull/443/files#diff-fad61aedc5779df83ed3dfea5f520c19e259eafc9e86bdf09447d7dab9706e65R18